gui/default/syncthing/core/syncthingController.js

```diff
@@ -1466,6 +1466,7 @@ angular.module('syncthing.core')
                 return;
             }
 
+	    var id = $scope.currentDevice.deviceID
             delete $scope.devices[id];
```

Comment:
Missing indentation. I'd directly put the expression as the key, without using the intermediate variable.

Reply:
Whitespace issue fixed, that was my editor inserting a tab for 8 spaces...

The id variable is used again twice inside the loop below, so I thought caching it locally helps readability.